### PR TITLE
Use fracoro data for all new UFS applications

### LINKED ISSUE #863: FIXfv3 not being set up correctly for atmos-only experiment in the latest workflow
Issue:  
An attempt to run  atmos-only experiment using the latest workflow (develop branch) and the model tag p8c revealed that FIXfv3 is not set up correctly for atmos-only runs if FRAC_GRID=.true..   The new VIIRS veg type datasets and new mask/oro datasets are only made available in the fix_fv3_fracoro directory. 

In the original [exglobal_forecast](https://github.com/yangfanglin/global-workflow/blob/feature/gfsv17/scripts/exglobal_forecast.sh).sh I have the following settings.
```
reso=025
[[ ${CASE} = "C192" ]] && reso=050
[[ ${CASE} = "C96" ]]  && reso=100
frac_grid=${frac_grid:-".true."}
if [ $frac_grid = ".false." ]; then
 FIXfv3=$FIX_DIR/fix_fv3_gmted2010/${CASE}
else
 FIXfv3=$FIX_DIR/fix_fv3_fracoro/${CASE}.mx${reso}_frac
fi
```
I think the latest fix, mask and oro datasets created by Shan/Mike/Helin should work for both fractional and non-fractional grid.   So the default FIXfv3 should be pointing to fix_fv3_fracoro instead of  fix_fv3_gmted2010 for all new UFS applications.   We need to use the new VIIRS veg type,  VIIRS based land-sea-lake mask and the updated oro dataset that was made to be consistent with MOM6, no matter if we are running coupled or atmos-only experiments.   

 fix_fv3_gmted2010 should only be used for running current operational model (GFS.v16)

----------------
Helin checked the model RT settings and found out the RTs are using the correct (new) datasets.

### PR DESCRIPTION
**Pull request by ChunxiZhang-NOAA**
**Description**

The new fracoro data should be used for all new UFS applications no matter if it uses frac_grid or not. 

Most problems in Issue[#863](https://github.com/NOAA-EMC/global-workflow/issues/863) have been resolved. However, one problem remains, e.g., the latest fix, mask and oro datasets (fracoro) created by Shan/Mike/Helin should work for both fractional and non-fractional grid. 

Related to the changes in UFS_UTILS. A PR[#714](https://github.com/ufs-community/UFS_UTILS/pull/741) in UFS_UTILS has been created. 

Fixes #863 

**Type of change**

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

**How Has This Been Tested?**

The Forecast-only tests were successful on Orion.   The tests are: Atmosphere-only C96 for FRAC_GRID on and off, respectively; Coupled c384 for GRAC_GRID on and off, respectively. However, the test for GLDAS is not done.

- [ ] Clone and Build tests on WCOSS Dell P3
- [ ] Cycled test on Orion
- [ ] Forecast-only test on Hera
  
**Checklist**
This PR requires an upstream UFS_UTILS PR[#741](https://github.com/ufs-community/UFS_UTILS/pull/741). 
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes need updates to the documentation. I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] New and existing tests pass with my changes
- [ ] Any dependent changes have been merged and published

### CODE REVIEW

**Comment by aerorahul** [comment]
@ChunxiZhang-NOAA @yangfanglin 

Following up on the discussion in the coupled modeling group meeting, since we are moving to using fractional grid, can we replace/remove all instances where non-fractional grid are used?  Basically, only have fractional grid option in the system.

```
❯❯❯ grep -r -B2 -A4 FRAC_GRID *
parm/config/config.fcst-#coupling settings
parm/config/config.fcst-export cplmode="nems_frac"
parm/config/config.fcst:if [[ "${FRAC_GRID:-".true."}" = ".false." ]]; then
parm/config/config.fcst-  export cplmode="nems_orig"
parm/config/config.fcst-fi
--
parm/config/config.base.emc.dyn:export FRAC_GRID=".true."
--
scripts/exgdas_atmos_gldas.sh:# Set FIXgldas subfolder based on FRAC_GRID value
scripts/exgdas_atmos_gldas.sh:if [[ "${FRAC_GRID:-".true."}" = ".true." ]] ; then
scripts/exgdas_atmos_gldas.sh-  ln -fs "${FIXgldas}/frac_grid/FIX_T${JCAP}" "${RUNDIR}/FIX"
scripts/exgdas_atmos_gldas.sh-else
scripts/exgdas_atmos_gldas.sh-  ln -fs "${FIXgldas}/nonfrac_grid/FIX_T${JCAP}" "${RUNDIR}/FIX"
scripts/exgdas_atmos_gldas.sh-fi
--
ush/parsing_namelists_FV3.sh:  frac_grid    = ${FRAC_GRID:-".true."}
```

Basically,
- we no longer need to define `FRAC_GRID` in `config.base`.
- we no longer need to have logic `if [ ${FRAC_GRID} ... ]; then` anywhere
- update `scripts/exgdas_atmos_gldas.sh` to remove the `if` logic for `FRAC_GRID`
- set `frac_grid = .true.` in the namelist

**Comment by ChunxiZhang-NOAA** [comment]
@aerorahul Good idea. I will remove FRAC_GRID conditions in the workflow as you suggested.

**Comment by WalterKolczynski-NOAA** [comment]
@ChunxiZhang-NOAA Should this still be a draft, or is it ready to review?

**Comment by ChunxiZhang-NOAA** [comment]
@WalterKolczynski-NOAA It is still a draft.

**Comment by ChunxiZhang-NOAA** [comment]
@aerorahul All applications will use the new fracoro data since this dataset can be used by both fractional and non-fractional grids. After a discussion with Fanglin, we will keep the option grid_frac, but all fix directories point to new fracoro dataset.

**Comment by aerorahul** [comment]
> @aerorahul All applications will use the new fracoro data since this dataset can be used by both fractional and non-fractional grids. After a discussion with Fanglin, we will keep the option grid_frac, but all fix directories point to new fracoro dataset.

That is fine.  
- All applications will use fractional grid datasets.  Non-fractional datasets will continue to reside on the systems, but will not be used.
- Applications will retain the `frac_grid = .true. | .false.` functionality via the `input.nml`

**Review by WalterKolczynski-NOAA — APPROVED** [review]

**Comment by WalterKolczynski-NOAA** [comment]
I'm ready to merge this once testing is confirmed to be complete (no testing is listed in the PR).

**Comment by ChunxiZhang-NOAA** [comment]
I updated the testing in "How Has This Been Tested?". Note that is PR required the upstream UFS_UTILS PR[#741](https://github.com/ufs-community/UFS_UTILS/pull/741).

**Review by WalterKolczynski-NOAA — APPROVED** [review]

**Comment by WalterKolczynski-NOAA** [comment]
@ChunxiZhang-NOAA did the UFS Utils PR need to be merged first for this to work? If so, I'll need to revert.

**Comment by ChunxiZhang-NOAA** [comment]
@WalterKolczynski-NOAA The UFS_UTILS PR needs to be merged first.

**Comment by WalterKolczynski-NOAA** [comment]
> @WalterKolczynski-NOAA The UFS_UTILS PR needs to be merged first.

@ChunxiZhang-NOAA Okay, please open the PR again and keep it as a draft until the UFS UTILS PR is merged and you have updated the PR to update the UFS UTILS hash.

Also, in the future, please be clearer when a PR depends on another PR to be merged first (use the words "Depends on ". "Related to" does not imply a dependency of this PR on that one.

**Comment by ChunxiZhang-NOAA** [comment]
@WalterKolczynski-NOAA Sorry for the confusion. I've created a new draft PR#[1242](https://github.com/NOAA-EMC/global-workflow/pull/1242).

**Pull request merged**
